pkg/controllers/vdb/replicatedupgrade_reconciler.go

```diff
@@ -51,6 +55,14 @@ func MakeReplicatedUpgradeReconciler(vdbrecon *VerticaDBReconciler, log logr.Log
 	}
 }
 
+const (
+	// Entries into the status.upgradeState.replicas array for each replica.
+	// Replica A has all of the primary subclusters and some secondary
+	// subclusters. Replica B has only secondary subclusters.
+	replicaA = 0
```

Comment:
Doesn't replicaGroup/replicaSet make more sense? My assumption is that replica=pod/node

Reply:
This is good feedback. I think I'll use replicaGroup. ReplicaSet is term used in k8s already, so may be confusing.